libethereum/BlockChain.cpp

```diff
@@ -582,59 +532,22 @@ pair<ImportResult, ImportRoute> BlockChain::attemptImport(bytes const& _block, O
 ImportRoute BlockChain::import(bytes const& _block, OverlayDB const& _db, bool _mustBeNew)
 {
 	// VERIFY: populates from the block and checks the block is internally coherent.
-	VerifiedBlockRef block;
-
-#if ETH_CATCH
-	try
-#endif
-	{
-		block = verifyBlock(&_block, m_onBad, ImportRequirements::OutOfOrderChecks);
-	}
-#if ETH_CATCH
-	catch (Exception& ex)
-	{
-//		clog(BlockChainNote) << "   Malformed block: " << diagnostic_information(ex);
-		ex << errinfo_phase(2);
```

Comment:
I haven't found any place where this additional info is used or at least printed out, so I think it's safe to remove this